unixfs/src/dir/builder/iter.rs

```diff
@@ -15,7 +15,7 @@ pub struct PostOrderIterator {
     block_buffer: Vec<u8>,
     // our stack of pending work
     pending: Vec<Visited>,
-    // "communication channel" from nested entries back to their parents this hashmap is only used
+    // "communication channel" from nested entries back to their parents this; hashmap is only used
```

Comment:
did I put the semicolon in the wrong spot :eyes:? I thought I put it after "parents"

Reply:
good spotting, yes you did ... I should had done it through the github ui :D retrying..